_posts/2016-03-31-Adding-TypeScript-to-Ionic-Framework.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+author: jan_de_wilde
+title: 'Adding TypeScript to Ionic Framework'
+image: /img/ionic-and-typescript.jpg
+tags: [TypeScript,Ionic]
+category: Ionic
+comments: true
+---
+
+### Ionic and TypeScript sitting in a tree
+
+So, [TypeScript](http://www.typescriptlang.org/) is the all new thing that allows you to use features from ES6 (or ES2015), ES7 and beyond. Say bye to loosely typed variables and say hello to modules, classes, interfaces and so much more.
+
+In order to use TypeScript in an Ionic Framework project there are a few small things you need to do to get things running.
+
+## 1. Install and configure the gulp package
+
+* Install the **gulp-tsc package** and save it to the development dependencies in `package.json`
+
+   {% highlight text %}npm install gulp-tsc --save-dev{% endhighlight %}
+
+* Next, require the package in your **gulpfile.js** like so
+
+   {% highlight text %}var typescript = require(‘gulp-tsc’);{% endhighlight %}
```

Comment:
`require(‘gulp-tsc’)`
Do the single quotes not pose any problem (Word single quote instead of regular single quote)?